pkgs/applications/audio/roc-toolkit/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, scons
+, python
+, ragel
+, gengetopt
+, libuv
+, libpulseaudio
+, sox
+}:
+
+stdenv.mkDerivation {
+  name = "roc-toolkit";
+
+  nativeBuildInputs = [ scons python ragel gengetopt ];
+  buildInputs = [ libuv libpulseaudio sox ];
+
+  src = fetchFromGitHub {
+    owner = "roc-streaming";
+    repo = "roc-toolkit";
+    rev = "c89687330bfce6f4dce79826f7a235b581f2b49d";
+    sha256 = "1j2gqfpsh0hrwrwvanv6b893b9pds93rv7bl7am2050j3jqmfjrw";
+  };
+
+  prefixKey = "--prefix=";
+
+  sconsFlags = [
+    "--jobs=1"
+    "--disable-libunwind"
+    "--disable-openfec"
+    "--disable-tests"
+    "--disable-doc"
+  ];
+
+  meta = with lib; {
+    description = "Toolkit for real-time audio streaming over the network";
+    homepage = "https://roc-streaming.org/";
+    license = licenses.mpl20;
+    platforms = platforms.linux ++ platforms.darwin;
```

`platforms.all` could be good too
```suggestion
    platforms = platforms.unix;
```